lib/dues_calculator.rb

```diff
@@ -9,16 +9,22 @@ def self.dues_per_competitor(country_iso2, base_entry_fee_lowest_denomination, c
   end
 
   def self.dues_per_competitor_in_usd(country_iso2, base_entry_fee_lowest_denomination, currency_code)
-    country_band = CountryBand.find_by(iso2: country_iso2)&.number
+    country_band = CountryBand.find_by(iso2: country_iso2)
+    country_band_detail = country_band&.active_country_band_detail
+    registration_fees_in_usd = Money.new(base_entry_fee_lowest_denomination, currency_code).exchange_to("USD")
 
-    input_money_us_dollars = Money.new(base_entry_fee_lowest_denomination, currency_code).exchange_to("USD")
+    return nil if country_band_detail.nil?
```

Comment:
If you move this statement one line higher, you can save yourself an extra call to `exchange_to` :wink:

Reply:
Okay, I thought of keeping inits at the beginning. I've moved the above line below now (same as you suggested). Hope it is fine now.